app/assets/stylesheets/layout/common/_contact_panels.scss

```diff
@@ -11,6 +12,21 @@
 .l-contact-panels--homepage {
   border-top: 0;
   margin-top: 0;
+
+  @include respond-to($mq-l) {
+    &:before, &:after {
+      content: "";
+      position: absolute;
+      border-right: 1px solid $color-newsletter-input-border;
+      height: 0;
+      height: 400px;
```

Comment:
Should there be two values for 'height', or should the 'height:0' declaration be outside the media queries?